digitalpublicgoods/tupaia.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "Tupaia",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.beyondessential.com.au/terms-and-conditions/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/beyondessential/tupaia"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "No",
+    "nonPIIAccessMechanism": ""
```

Comment:
Tupaia needs to clarify how the mechanism for extracting non-PII data into a non-proprietary format occurs. Thanks.